test/ch/imedias/rsccfx/model/xml/SupporterHelperTest.java

```diff
@@ -0,0 +1,146 @@
+        + "        <chargeable>false</chargeable>\n"
+        + "        <description>imedias</description>\n"
+        + "        <encrypted>false</encrypted>\n"
+        + "        <port>5500</port>\n"
+        + "    </supporter>\n"
+        + "    <supporter>\n"
+        + "        <address>agora.imedias.ch</address>\n"
+        + "        <chargeable>false</chargeable>\n"
+        + "        <description>imedias (encrypted)</description>\n"
+        + "        <encrypted>true</encrypted>\n"
+        + "        <port>50000</port>\n"
+        + "    </supporter>\n"
+        + "</supporters>\n";
+
+
+    supporter1 = new Supporter("imedias", "agora.imedias.ch", "5500",
+        false, false);
+    supporter2 = new Supporter("imedias (encrypted)", "agora.imedias.ch", "50000",
+        true, false);
+    expectedSupporters = new ArrayList<>();
+    expectedSupporters.add(supporter1);
+    expectedSupporters.add(supporter2);
+    supporterLast = new Supporter();
+  }
+
+  /**
+   * Test for {@link SupporterHelper#loadSupporters()}.
+   */
+  @Test
+  public void testLoadSupporters() throws Exception {
+    // if preferences are null, test if default supporters are returned
+    supporterHelper.setSupportersInPreferences(null);
+    List<Supporter> actualSupporters = supporterHelper.loadSupporters();
+    verifyDefaultSupporters(actualSupporters);
+    actualSupporters = null;
+
+    // if preferences are not null
+    supporterHelper.setSupportersInPreferences(supportersXml);
+    actualSupporters = supporterHelper.loadSupporters();
+    assertEquals(2, actualSupporters.size());
+    assertEquals(supporter1, actualSupporters.get(0));
+    assertEquals(supporter2, actualSupporters.get(1));
+    actualSupporters = null;
+
+    // if preferences contain invalid xml
+    String invalidXml = "<test>this is not a correctly formatted xml</not test>";
+    supporterHelper.setSupportersInPreferences(invalidXml);
+    actualSupporters = supporterHelper.loadSupporters();
+    verifyDefaultSupporters(actualSupporters);
+  }
+
+  /**
+   * Test for {@link SupporterHelper#saveSupporters(List)}.
+   */
+  @Test
+  public void testSaveSupporters() throws Exception {
+    // standard case
+    supporterHelper.setSupportersInPreferences(null);
+    supporterHelper.saveSupporters(expectedSupporters);
+    assertEquals(supportersXml, supporterHelper.getSupportersXmlFromPreferences());
+  }
+
+  /**
+   * Test for {@link SupporterHelper#getDefaultSupporters()}.
+   */
+  @Test
+  public void testGetDefaultSupporters() throws Exception {
+    List<Supporter> actualSupporters = supporterHelper.getDefaultSupporters();
+    verifyDefaultSupporters(actualSupporters);
+  }
+
+  /**
+   * Verifies that the supporters list is actually the list of default supporters.
+   * Not marked with a @Test annotation because it is indirectly called in other tests.
+   */
+  private void verifyDefaultSupporters(List<Supporter> actualSupporters) {
```

Comment:
If you're sticking with the non-test-but-somehow-test-anyway thing, I would move this at the end or at the beginning of the file. It is quite easy to not get that "@test" is missing while scrolling through